test.js

```diff
@@ -160,8 +160,8 @@ describe('Tests mongoose-timezone-offset', function() {
     });
 
     describe('edge cases', function() {
-        it('Should not break with bad offset of 17', async function() {
```

Comment:
this is not so good because you did not return a promise
see 
<p>An <code>async</code> function always wraps the return value in a <code>Promise</code>. Using <code>return await</code> is therefore redundant.</p>
<h2>Noncompliant Code Example</h2>
<pre>
async function foo() {
  // ...
}

async function bar() {
  // ...
  return await foo(); // Noncompliant
}
</pre>
<h2>Compliant Solution</h2>
<pre>
async function foo() {
  // ...
}

async function bar() {
  // ...
  return foo();
}
</pre>



---
**Sema Reaction:** :ok_hand: Looks good | **Sema Tags:** None